### ISSUE TITLE
Unofficial artifacts failing when depending on sister subprojects

### ISSUE THREAD

**Issue by stonier** [rosjava/rosjava_bootstrap#12]
The unofficial artifact generator code for custom messages is trying to depend on external artifacts. This is ok for single projects. But fails is you start filling up a project with message artifact generating subprojects.

**Comment by stonier** [comment]
Key problem is here in the catkin plugin:

```
  messageDependencies().each { d ->
            p.dependencies.add("compile", 'org.ros.rosjava_messages:' + d + ':[0.1,)')
        }
```

This is adding an external dependency, which will fail if you need to look inside subprojects in the same build.

**Comment by stonier** [comment]
Bugfixed for hydro in 6368c64d4542edb66084c6ad470fa59994c09ab4, released debs and maven artifacts for v0.1.13.
